SignalRServiceBenchmarkPlugin/utils/Commander/ArgsOption.cs

```diff
@@ -0,0 +1,74 @@
+﻿using CommandLine;
+using Common;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Commander
+{
+    // TODO: edit descriptions and default values
+    public class ArgsOption
+    {
+        // App server
+        [Option("AppserverPort", Required = false, Default = 5050, HelpText = "Port of app server.")]
+        public int AppserverPort { get; set; }
+
+        // Remote
+        [Option("RpcPort", Required = false, Default = 5050, HelpText = "Port to be conencted from remote hosts.")]
```

Comment:
Default RpcPort and AppServerPort should be different.

Reply:
Ok, thanks... I forget to change the default value.